tests/cli/test_datasets.py

```diff
@@ -602,6 +602,42 @@ def test_datasets_ls_files_check_exit_code(output_format, runner, project):
     assert 0 == result.exit_code
 
 
+def test_datasets_ls_files_lfs(tmpdir, runner, project):
+    """Test file listing lfs status."""
+    # NOTE: create a dataset
+    result = runner.invoke(cli, ["dataset", "create", "my-dataset"])
+    assert 0 == result.exit_code
+    assert "OK" in result.output
+
+    # NOTE: create some data
+    paths = []
+
+    new_file = tmpdir.join("file_1")
+    new_file.write(str(1))
+    paths.append(str(new_file))
+
+    new_file = tmpdir.join("file_2")
+    new_file.write(str(2) * 200000)
+    paths.append(str(new_file))
```

Comment:
There is also a `large_file` fixture where you can use for this purpose.

Reply:
Thanks, I didn't know about that one!